spec/services/ncr/reporter_spec.rb

```diff
@@ -21,40 +23,27 @@
     end
   end
 
-  describe ".proposals_tier_one_pending" do
+  describe "#proposals_tier_one_pending" do
     it "only returns Proposals where Tier One approval is actionable" do
```

Comment:
Did you reduce this one because we no longer have/use Tier 1 & 2 approvers with the new policy, or for another reason?

(And yeah, I suspect we need to check for test breakages with the policy changes)